pkg/minikube/registry/drvs/hyperkit/hyperkit.go

```diff
@@ -87,7 +88,11 @@ func status() registry.State {
 		return registry.State{Error: err, Fix: "Run 'brew install hyperkit'", Doc: docURL}
 	}
 
-	cmd := exec.Command(path, "-v")
+	// Allow no more than 2 seconds for querying state
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
```

Comment:
some times docker takes more than 2 secs, how about 3 ? in honor of three "good algorithm, good syntax, good e2e test"

Reply:
I was going to send the PR in as 1 second, because it seemed to be a good hard limit for auto-detection. Asking a user to wait 3 seconds for a bad hypervisor they are not intending to use is asking too much.

To make e2e stable, I doubled it. None of these commands should ever take >250ms to execute. If they do, we're running the wrong command.

`docker info` reliably takes 0.10s on Linux on macOS, so the timeout is already 20x what normal should be. I'd be happy to raise it if we see evidence of it causing flakiness, but I would aim for less time (1.5s?) rather than more time spent in autodetection.